CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Changed
+- Added deprecation note
```

Comment:
Just as a tip: I usually hear more the term `Deprecation Notice` _(years of PHP taught me that)_

Reply:
Nice one! Thank you for letting me know! 🚀